sycl/include/sycl/ext/oneapi/matrix/matrix-tensorcore.hpp

```diff
@@ -1,8 +1,5 @@
 #pragma once
```

Comment:
Please, add a comment with license reference. Similar to this one:
```c++
//==---------------- sycl.hpp - SYCL standard header file ------------------==//
//
// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
// See https://llvm.org/LICENSE.txt for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
//
//===----------------------------------------------------------------------===//
```

Reply:
Ah yes, thank you for pointing this out.  I noticed that the corresponding reference in matrix-jit.hpp is labelled matrix.hpp and matrix-aot-amx.hpp is labelled matrix-amx.hpp.  Should I correct these also while I am at it?

Reply:
Yes, please.

Reply:
Done

Reply:
Thank you for working on this!